### ISSUE TITLE
TouchEffect.Command and TouchEffect.LongPressCommand in CollectionView Doesn't work together with TalkBack (Xamarin.Forms)

### ISSUE THREAD

**Issue by vsfeedback** [xamarin/Xamarin.Forms#15654]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/TouchEffectCommand-and-TouchEffectLong/10238298)._

---
[severity:I'm unable to use this version]
If you want to support long press on items in a CollectionView you need to use both TouchEffect.LongPressCommand and TouchEffect.Command in the DataTemplate.
However when you do this instead of using the SelectionChanged event you can't execute a long press nor a tap with TalkBack on Android. If you turn off TalkBack and interact with the CollectionView without the screen reader both gestures works as expected.

Define the CollectionView as follows:

                    
                        
                            
                                
                                    <label>
                                
                            
                        

---
#### Feedback Bot on 12/25/2022, 06:35 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)
